WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaPickerFragment.kt

```diff
@@ -343,36 +363,41 @@ class MediaPickerFragment : Fragment() {
     private fun setupPhotoList(uiModel: PhotoListUiModel) {
         when (uiModel) {
             is Data -> {
-                recycler.setEmptyViewIfNull(actionable_empty_view)
-                if (recycler.adapter == null) {
-                    recycler.adapter = MediaPickerAdapter(
-                            imageManager
-                    )
-                }
-                val adapter = recycler.adapter as MediaPickerAdapter
-
-                (recycler.layoutManager as? GridLayoutManager)?.spanSizeLookup =
-                        object : GridLayoutManager.SpanSizeLookup() {
-                            override fun getSpanSize(position: Int) = if (uiModel.items[position].fullWidthItem) {
-                                NUM_COLUMNS
-                            } else {
-                                1
-                            }
-                        }
-                val recyclerViewState = recycler.layoutManager?.onSaveInstanceState()
-                adapter.loadData(uiModel.items)
-                recycler.layoutManager?.onRestoreInstanceState(recyclerViewState)
```

Comment:
I did it on purpose because I don't think it's necessary (it was one of the attempts to fix the crash). I don't think the scroll position is updated when we use the diff utils adapter. What do you think?